mesonbuild/modules/i18n.py

```diff
@@ -390,6 +392,7 @@ def itstool_join(self, state: 'ModuleState', args: T.List['TYPE_var'], kwargs: '
             install=kwargs['install'],
             install_dir=[kwargs['install_dir']] if kwargs['install_dir'] is not None else None,
             install_tag=install_tag,
+            description='Processing translations from {}',
```

Comment:
itstool_join is an advanced alternative to msgfmt as used with merge_file().

Reply:
so what should I change?

Reply:
It is merging translations for the output file, so it too should be "Merging translations for"

Reply:
for, not from :)

Reply:
oups! :-S